MIPS/mip-0006-slot-reduction-90s.md

```diff
@@ -0,0 +1,270 @@
+    - Currently set to 24 by default
+    - Not constrained by the protocol, but will be removed completely
+
+## Prerequisites
+
+This update requires a prerequisite improvement to the SNARK work production code, which will be delivered before the hard fork as part of a soft fork update. This improvement will allow coordinators to use workers more efficiently by giving them smaller pieces for proving. See Appendix for more details.
+
+After delivering that update, the network will require at least some of the community members running the SNARK workers to run multiple (at least 4) SNARK workers per coordinator to ensure timely processing of SNARK work for the heaviest transactions.
+
+# Rationale
+
+The primary rationale for this change is to increase the frequency of block production (throughput) in the Mina blockchain while maintaining the economic model. Recent performance improvements make this change viable without sacrificing system stability. 
+
+Additionally, this change will improve latency, resulting in confirmation times for transactions cut in half and enhancing overall network responsiveness.
+
+The specific value of 90 seconds was chosen as it:
+
+1. Doubles the potential throughput of the network
+2. Remains safely within the performance capabilities of the improved system
+
+# Backwards Compatibility
+
+This MIP is not backward compatible as it changes consensus rules and requires a hard fork to implement. It is explicitly incompatible with the current "Berkeley" release and must be implemented in the next hard fork.
+
+This MIP doesn't require any changes to the protocol beyond those mentioned in the Specification section. There is no specific hard fork delivery strategy required for implementing this change.
+
+# Security Considerations
+
+The reduction in slot time could potentially impact network security if nodes are unable to keep up with the increased rate of block production. However, the recent performance improvements mitigate this risk.
+
+Some security considerations include:
+
+- Network propagation time for blocks becomes more critical with shorter slots
+- Node synchronization may occur more frequently
+- Block production failures could increase if hardware cannot keep pace
+- Some blocks may contain fewer transactions than they could (or worst-case zero transactions, only coinbase) if SNARK work production for the heaviest transactions is lagging behind
+
+Testing on various hardware configurations prior to implementation was a necessary step to ensure network stability at the new slot time.
+
+The removal of the zkapp soft limit configuration option will increase the RAM consumption of Mina nodes. However, recently implemented [RAM optimizations](https://github.com/MinaProtocol/mina/pull/16966) ensure that RAM usage after this removal remains well below the recommended [RAM specifications](https://docs.minaprotocol.com/berkeley-upgrade/requirements) for Mina node deployment.
+
```

Comment:
Consider adding community impacts and identify any gating concerns, for example:
Explorers may need time to adjust to epoch times
Payouts processes may need to be adjusted